lib/scrape.js

```diff
@@ -165,6 +166,45 @@ async function findOrganization({
   return null
 }
 
+async function findWiki(name) {
+  const APIurl = wdk.searchEntities({
+    search: name,
+    format: 'json',
+  })
+  const srch = await fetch(APIurl)
+  const response = await srch.json()
+  if (response && response.search.length) {
+    const wikidataurl = response.search[0].url
+    const wikidataid = response.search[0].id
+    const entityAPIurl = wdk.getEntities({
+      ids: [wikidataid],
+      props: ['sitelinks'],
+      format: 'json',
+    })
+
+    const sitelinks = await fetch(entityAPIurl)
+    let sitelink = await sitelinks.json()
+    sitelink = sitelink['entities'][wikidataid]['sitelinks']
+    if (sitelink && sitelink['enwiki']) {
+      const wikipediaurl = wdk.getSitelinkUrl(sitelink['enwiki'])
+      return {
+        url: wikipediaurl,
+        type: 'wikipedia',
+      }
+    } else {
+      return {
+        url: 'https:' + wikidataurl,
```

Comment:
This is probably not needed because `//` is actually totally valid for a URL -- it's protocol aware so it uses either `http` or `https`  depending on what the browser is using (though I could totally see wanting to do it this way out of consistency with other URLs).

Reply:
Yep, but wikipedia/wikidata is guaranteed to provide HTTPS, not so much guaranteed for the gci-leaders website, I just think it's better to default to HTTPS?

Reply:
Sure that works!

Reply:
Maybe in that case `https:${wikidataurl}`?

Reply:
I've decided to remove `https:`, since I just noticed netlify also does https, so there's no point to it then. Will put it back if anyone thinks it's a problem.

Reply:
FYI, netlify without custom domain doesn't have HTTPS enforcement, but this shouldn't be a problem at all.